docs/framework/windows-workflow-foundation/toc.yml

```diff
@@ -7,8 +7,11 @@
     href: whats-new.md
   - name: What's New in Windows Workflow Foundation in .NET 4.5
     href: whats-new-in-wf-in-dotnet.md
-  - name: Deprecated types in Windows Workflow Foundation
+  - name: Obsolete types and tools
```

Comment:
We can't have nodes that are both links to a file and expander nodes (it's for accessibility). So it needs to look as follows:

```
  - name: Deprecated types
    items:
    - name: Obsolete types and tools
      href: deprecated-types.md
    - name: "wfc.exe - Workflow Compiler Tool"
      href: "wfc-exe-workflow-compiler-tool.md" 
```

Reply:
look at Getting started section, build failed because name with no href.
```
  - name: Getting Started Tutorial
    href: getting-started-tutorial.md
    items:
    - name: "How to: Create an Activity"
      href: how-to-create-an-activity.md
    - name: "How to: Create a Workflow"
      href: how-to-create-a-workflow.md
```

Reply:
I think it failed on an earlier commit because an href had a hyphen in front of it. You *can* enter it as it's done for Getting Started Tutorial, but it renders like this (note the duplicated heading):

![image](https://user-images.githubusercontent.com/24882762/95141707-badb0680-0726-11eb-9af7-36a69a1411b3.png)

This is what the [contributor guide](https://review.docs.microsoft.com/en-us/help/contribute/contribute-toc-mvc-how-to?branch=master#toc-labels-and-links) says:

"Parent nodes should expand and not be a link."

Reply:
toc.xml looks right to me after fixing the build error and warnings.  Can you please review, @gewarren ?

There are two remaining warnings, one is complaining about H1 of deprecatecontent-md.md file, the second is about the config section should be treated like code.  Should we really need to fix these two warnings?

Reply:
Yes, all the build warnings will need to be resolved. I'll review and leave comments/suggestions about how to fix them.

Reply:
Thanks @gewarren !  All checks have passed now.